private-templates-service/adaptivecards-templating-service/src/authproviders/AzureADProvider.ts

```diff
@@ -0,0 +1,57 @@
+import { AuthenticationProvider } from "./IAuthenticationProvider";
+import jws, { Algorithm, Signature } from "jws";
+import axios from "axios";
+
+/**
+ * @class
+ * Class representing authentication with Azure AD
+ * @extends AuthenticationProvider
+ */
+export class AzureADProvider implements AuthenticationProvider {
+    private user : string = "";
+    private static CERT_URL : string = "https://login.microsoftonline.com/common/discovery/keys";
+
+    async isValid(accessToken: string) : Promise<boolean> {
+        let bearer = accessToken.split(/[ ]+/).pop();
+        if (bearer){
+            accessToken = bearer;
+        }
+
+        let decodedToken : Signature = jws.decode(accessToken);
```

Comment:
nit: but the formatting of all of your " variablename : type" is inconsistent with the other PRs.  Should try to align your auto formatting

Reply:
Fixed!